connectors/http-connector/pom.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.pacesys.openstack4j.connectors</groupId>
+        <artifactId>openstack4j-connectors</artifactId>
+        <version>2.0.2-SNAPSHOT</version>
+    </parent>
+    <name>OpenStack4j HttpURL Connector</name>
+    <artifactId>openstack4j-http-connector</artifactId>
+    <packaging>jar</packaging>
```

Comment:
Travis CI is failing and local tests too since the maven bundle plugin wasn't added.  Look at another connector and copy in the <build> .. </build> tag replacing the package info